src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundAssignmentTests.cs

```diff
@@ -1194,27 +1239,27 @@ int M(int i)
 
         [Theory, WorkItem(38054, "https://github.com/dotnet/roslyn/issues/53969")]
         [InlineData(
-            "/* Before */ i [||]= i + 1; /* After */",
+            "/* Before */ i [|=|] i + 1; /* After */",
             "/* Before */ i++; /* After */")]
         [InlineData(
-            "M( /* Before */ i [||]= i + 1 /* After */ );",
+            "M( /* Before */ i [|=|] i + 1 /* After */ );",
             "M( /* Before */ ++i /* After */ );")]
         [InlineData(
-            "M( /* Before */ i [||]= i - 1 /* After */ );",
+            "M( /* Before */ i [|=|] i - 1 /* After */ );",
             "M( /* Before */ --i /* After */ );")]
         public async Task TestTriviaPreserved(string before, string after)
         {
-            await TestInRegularAndScript1Async(
+            await VerifyCS.VerifyCodeFixAsync(
 @$"public class C
 {{
-    int M(int i)
+    void M(int i)
```

Comment:
Not a fan of changing test content

Reply:
Well, it is better to change 1 word in test than to add compiler error diagnostic, which is not the main point of the test, right?

Reply:
If you are going to change the tests, please do so in another commit. Doing so in the same commit just increases the cost and complexity of reviewing here. Thanks! :-)